### DISCUSSION TITLE
Created new Facebook patches to implement

### DISCUSSION THREAD

**Discussion by PantlessCoding** [ReVanced/revanced-discussions#1069] [Question] [answered]
Hello, 
I experimented with decompiling and reversing APKs and I created a mod to fully hide sponsored Stories on the main feed. Also made another one to disable auto-play on videos regardless of settings.

I have the smali changes required, but it would be cool to make this a revanced patch. I never done one either, but I am interested in doing it myself. Whould this be something that could be added in the main repos if done ?

I have setup a new repo from the template, not sure if it is the correct way of contributing.

Thanks !

**Comment by oSumAtrIX** [comment, marked as answer]
Hi, the template repo is in case you want to maintain your own set of patches. If you want to contribute them to ReVanced, you can fork the patches repository and PR it. The revanced-documentation repo is ideal for new developer's to learn how to develop ReVanced patches

**Reply by PantlessCoding** [reply]
Thanks, will do.
